broker/lib/controllers/ServiceBrokerApiController.js

```diff
@@ -191,19 +362,49 @@ class ServiceBrokerApiController extends FabrikBaseController {
       .catch(ServiceBindingAlreadyExists, conflict);
   }
 
+  putBindingV2(req, res) {
```

Comment:
what is the difference between this code and the v1 method code? Since its not different, can you please merge it to one? Also in this case are you not putting in a read lock on this operation?

Reply:
Removed duplicate methods.
For bind/unbind operations we only check if deployment is WRITE locked. No lock is put as it is a sync operation